### PR TITLE
Bug 2106736: Add multiplePVsSameID capability

### PR DESCRIPTION
**Pull request by jsafrane**
This CSI driver can handle multiple PVs that have the same VolumeHandle used on the same node.

cc @openshift/storage

### CODE REVIEW

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Review by bertinatto — COMMENTED** [review]
/lgtm

File: test/e2e/manifest.yaml [hunk 1]
```diff
@@ -32,3 +32,4 @@ Timeouts:
   PodStart: 15m
   PodDelete: 15m
   PVDelete: 20m
+    multiplePVsSameID: true
```
Comment (duanwei33) [inline test/e2e/manifest.yaml:35]: Is it mandatory to put it under Capabilities?

Comment (jsafrane) [reply test/e2e/manifest.yaml]: Good catch! My script was too stupid. Fixed.

**Pull request merged**